README.md

```diff
@@ -96,12 +94,11 @@ make clean
 
 ### Clone
 
-To clone Gpfsbeat from the git repository, run the following commands:
+To clone {Beat} from the git repository, run the following commands:
 
 ```
-mkdir -p ${GOPATH}/github.com/hpcugent
-cd ${GOPATH}/github.com/hpcugent
-git clone https://github.com/hpcugent/gpfsbeat
+mkdir -p ${GOPATH}/src/github.com/itkovian/gpfsbeat
```

Comment:
@itkovian why all the changes from hpcugent to itkovian? especially because this is an hpcugent repo?